forge-gui/res/cardsfolder/upcoming/wreck_hunter.txt

```diff
@@ -0,0 +1,12 @@
+Name:Wreck Hunter
+ManaCost:B B
+Types:Creature Human Artificer
+PT:2/2
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChoosePlayer | TriggerDescription$ When CARDNAME enters the battlefield, choose target player. You create a tapped Powerstone token for each nonland card in that player's graveyard that was put there from the battlefield this turn. (It's an artifact with "{T}: Add {C}. This mana can't be spent to cast a nonartifact spell.")
+SVar:TrigChoosePlayer:DB$ ChoosePlayer | Defined$ You | ValidTgts$ Player | Choices$ ThisTargetedPlayer | SubAbility$ DBToken
```

Comment:
Might as well get rid of the Defined$, that's just asking for trouble in the future